script/Ciel/Day 8/08_01_CIEL08_1D.txt

```diff
@@ -419,7 +419,7 @@ My consciousness grows smaller and smaller.
 [sha:7632bba247b4373abbb0369f0ea3cd6efe044a57]{
 -- Page 125, Offset 18581. Mods: @n.
 -- 俺という個が、自らの低級さに耐えられず<萎|しぼ>んでいく。
-My existence as an individual begins to waste away, unable to bear its own vulgarity.
+My individual self begins to waste away, unable to bear its own vulgarity.
```

Comment:
Prefer original, or possibly 'sense of self'

Reply:
"individual" is the keyword, so that's not changing.

Maybe just individuality...

Reply:
In the simplest sense it's "The individual that is me" (boo, boooring)
Maybe "the individual that comprises me" would sound better.

Reply:
Is individual _that_ important? That construction is somewhat formal / unnatural. Self, ego, etc. lend themselves to more fluid phrasing.

Reply:
Ego is already used on the line above.
Self on the next one.
Then there's the repetition of 'my' (3 times) which was good to keep intact.

Reply:
Welp in that case I think the original has my vote still